src/components/Account/index.module.css

```diff
@@ -9,6 +9,7 @@
   border-radius: 12px;
   background: var(--white);
   padding: 0.7125rem;
+  text-display: none;
```

Comment:
I don't think this is a valid css property?

Reply:
Oh right. It was supposed to be `text-decoration`. Fixed.